guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -0,0 +1,33 @@
+[id="requirements-for-installation-in-an-ipv6-network_{context}"]
+= Requirements for {Project} Installation in an IPv6 Network 
+
+Before installing {Project} in an IPv6 network, ensure that you meet the following requirements:
+
+ifeval::["{build}" == "satellite"]
+* {SmartProxy} must be installed on {RHEL} version 7.9 or higher.
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+* {SmartProxy} must be installed on a system with grub2 version 2.05 or higher or system with fixes for HTTP Boot, for example CentOS version 7.9 or 8.3 or higher. For other operating systems, copy the newest grub build to `/var/lib/tftpboot/grub2/grubx64.efi`.
+endif::[]
+
+* You can install {Project} and {SmartProxy} in IPv6-only systems, dual-stack installation not supported.
+
+* It is assumed, native IPv6 internet connectivity is available. If not, it's recommended to configure a HTTP proxy in order to download RPM packages:
```

Comment:
Is not this the same thing as in this step in guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc within this PR?

> Optional: If native IPv6 connectivity is not available, configure an HTTP proxy in {Project} and set it as the default proxy in settings. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8-beta/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].

Reply:
Yes it is just a generic info about how we recommend to setup proxy when native IPv6 is not there. Feel free to make this a module.